MovieApp/MovieApp/App/Features/Movies/ViewControllers/DetailsViewController/DetailsPresenter.swift

```diff
@@ -9,7 +9,12 @@ class DetailsPresenter {
     }
     
     func fetchData() {
-        delegate?.setData(with: MovieDetailsViewModel())
+        let data = MovieDetailsViewModel()
+        
+        let infoData: MainInfoViewModel = data.info
+        let overviewData: OverviewViewModel = data.overview
+        let actorsData: ActorsViewModel = data.actors
+        delegate?.setData(infoData: infoData, overviewData: overviewData, actorsData: actorsData)
```

Comment:
why do you need separate variables for setData func?

Reply:
I thought it would be good to remove that logic from ViewController and give ViewController concrete variables.

Is it overkill?